plugins/janus_streaming.c

```diff
@@ -2222,7 +2371,8 @@ struct janus_plugin_result *janus_streaming_handle_message(janus_plugin_session
 			/* The category to remove is the mountpoint name */
 			janus_config_remove_category(config, mp->name);
 			/* Save modified configuration */
-			janus_config_save(config, config_folder, JANUS_STREAMING_PACKAGE);
+			if(janus_config_save(config, config_folder, JANUS_STREAMING_PACKAGE) < 0)
+				save = FALSE;	/* This will notify the user the mountpoint is not permanent */
```

Comment:
This looks like a good suggestion, it might be worth adding this to other plugins as well (and to the other potentially permanent requests in this plugin).

Reply:
Yea, I wasn't sure, I figure you'd notice and make the call! 👍